components/app.jsx

```diff
@@ -7,6 +7,7 @@ import { BrowserRouter } from 'react-router-dom';
 import { RequireAuth, AuthConsumer, AuthProvider } from './auth.jsx';
 import { Routes, Route } from 'react-router-dom';
 import { createRoot } from 'react-dom/client';
+import SingleEvent from './singleEvent.jsx';
```

Comment:
I don't feel _too_ strongly about this but I feel like this component should just be named `Event`. Since the index component is called `Events` it seems logical that the show component is the singular.

Reply:
I agree! Good point. I'll change that. Thank you!